frontend/src/enums/CanvasMode.ts

```diff
@@ -0,0 +1,5 @@
+export enum CanvasMode {
+  Selection = "selection-mode",
+  SimpleDrawing = "drawing-mode",
+  Eraser = "eraser-mode",
+}
```

Comment:
I'm not sure why we decided to name enums with small letters, but I would much rather prefer capital letter. So it would be 
```suggestion
export enum CanvasMode {
  SELECTION = "selection-mode",
  SIMPLE_DRAWING = "drawing-mode",
  ERASER = "eraser-mode",
}
```

It's the same case for all the enums